.gitignore

```diff
@@ -1,4 +1,5 @@
 .build-docker
+.build.octopus
```

Comment:
This is generated by the `Makefile`. You should probably use `.build.*` so that the file for other versions gets ignored too.

Reply:
Make sense!. Changing the code